eventhubs/client/azure-eventhubs/src/main/java/com/azure/messaging/eventhubs/EventHubConsumer.java

```diff
@@ -20,27 +21,37 @@
 import static com.azure.messaging.eventhubs.EventHubConsumerOptions.MINIMUM_PREFETCH_COUNT;
 
 /**
- * This is a logical representation of receiving from an Event Hub partition.
- *
- * <p>
- * A {@link EventHubConsumer#receive()} is tied to a Event Hub partitionId + consumer group combination.
+ * A consumer responsible for reading {@link EventData} from a specific Event Hub partition and as a member of a
```

Comment:
For what it's worth, I ended up changing my version of this to "... from a specific Event Hub partition in the context of a specific consumer group."